### DISCUSSION TITLE
Test fattura elettronica

### DISCUSSION THREAD

**Discussion by andyweb** [fattureincloud/api#403] [Q&A] [answered]
Ciao, come faccio a testare la creazione di una fattura elettronica?

**Comment by emajo** [comment, marked as answer]
Ciao @andyweb, può trovare una guida per la creazione di una fattura [qui](https://developers.fattureincloud.it/docs/guides/invoice-creation) e un altra guida sulla gestione della fattura elettronica [qui](https://developers.fattureincloud.it/docs/guides/e-invoice-management).

Grazie,
Emanuele.

**Reply by valmoz** [reply]
Buongiorno @andyweb possiamo chiudere?

**Comment by andyweb** [comment]
Si, grazie per il momento.

Il giorno mer 31 gen 2024 alle ore 11:48 Mauro Valota <
***@***.***> ha scritto:

> Buongiorno @andyweb <https://github.com/andyweb> possiamo chiudere?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/fattureincloud/api/discussions/403#discussioncomment-8317891>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA5NHAMOJIZ3ME5PN65K6QDYRIORPAVCNFSM6AAAAABCL3JPFSVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DGMJXHA4TC>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

WIM di Andrea Spurio
62026 San Ginesio (MC)
mobile: 329 8530293
www.italiawim.it
